### PR TITLE
Feature/sponsors

### PR DESCRIPTION
**Pull request by mackenly**
Drafts a solution to #8 

- Implements a sponsors.json file 
- Adds light and dark versions of sponsor logos
- Creates a Sponsors component that shows sponsor logos

### CODE REVIEW

File: src/components/links.svelte [hunk 1]
```diff
@@ -1,9 +1,3 @@
-<script lang="ts">
```
Comment (mackenly) [inline src/components/links.svelte:1]: This was something I had forgotten to take out from a svelte doc example in a previous PR. It doesn't do anything for us.

**Comment by fudgemasterultra** [comment]
It looks pretty good! What do you think of adding in arrows to let the user change to the next sponsor more quickly, or go back to previous one?

**Comment by RyannosaurusRex** [comment]
I like @fudgemasterultra's suggestion of a button to change sponsors forward and back, but it looks great in line with everything else!

**Comment by RyannosaurusRex** [comment]
Uh oh, this exposes a bug I didn't know we had. On iOS I'm unable to scroll. So this makes our content tall enough to cut off the top and bottom of the screen.  Might be a good follow-up bugfix, though?

**Comment by mackenly** [comment]
I'll take a look at adding arrows. Should be easy enough.

**Comment by mackenly** [comment]
> Uh oh, this exposes a bug I didn't know we had. On iOS I'm unable to scroll. So this makes our content tall enough to cut off the top and bottom of the screen.  Might be a good follow-up bugfix, though?

@RyannosaurusRex Do you want me to fix mobile issues in this one or will that be seperate?

**Pull request merged**